cfme/services/requests.py

```diff
@@ -221,9 +221,14 @@ def edit_request(self, values, cancel=False):
         view.flash.assert_no_error()
 
 
+class RequestsToolbar(View):
```

Comment:
:+1:

Reply:
MyService kinda needs this too, but I was trying to keep scope small. Only added for details, since there were 5 other widgets to change for the collection view toolbar.